### DISCUSSION TITLE
Starsat support

### DISCUSSION THREAD

**Discussion by MohamedAliClai** [PredatH0r/ChanSort#297] [Ideas]
Hi
star sat receivers chunnels is .sdx files
so any idea how can we edit this channels list with this app
or you planned to add such support

**Comment by PredatH0r** [comment]
ChanSort already supports 3 versions of .sdx files. This file format comes from SatcoDX, a OEM that provides the internal tuner hard- and software to man y different brands, including dedicated satellite receiver boxes and low cost TVs with internal satellite receivers.
If your particular file can't be opened with ChanSort, please send it to me (horst @ beham . biz) and I can take a look.
